### DISCUSSION TITLE
(MacOS) How to add foundryup to zsh

### DISCUSSION THREAD

**Discussion by devjvla** [Cyfrin/foundry-full-course-cu#1889] [Ideas]
1. Open your terminal
2. Type `code ~/.zshrc` *this will open .zshrc in vscode
3. Paste`source /Users/jovicabengona/.zshenv` at the very first line (or anywhere, just don't erase anything). Then save
4. Kill/Open terminal in vscode
5. You should be able to use `forge --version` etc.

**Comment by vigneshbalu96** [comment]
Great :100:

**Reply by EngrPips** [reply]
Indeed
